cms/cms-import.md

```diff
@@ -0,0 +1,275 @@
+
+<a name='record_type'></a>
+
+#### `record_type`
+The database table this import is linked to.
+```yml
+imports:
+  import-name-random: 
+    record_type: 
+```
+
+<a name='identifier'></a>
+
+#### `identifier`
+The field that Skygear uses to identify a record. If `identifier` is not provided, Skygear will use `_id` to identify records. 
+
+If a record in the database is found to have the same identifier as the imported record, Skygear will update the record in the database.
+
+```yml
+imports:
+  import-name-random: 
+    identifier: 
+```
+<a name='handle_duplicated_identifier'></a>
+
+#### `handle_duplicated_identifier`
+The way to handle duplicated records when If more than 1 record in the database is found to have the same identifier as the imported record.
+
+```yml
+imports:
+  import-name-random: 
+    handle_duplicated_identifier: throw-error or use-first
+```
+
+- `throw-error` will return an error about this record. If you want to reject the whole import when there are errors, you can set `atomic: true` in the `action` setting (will be explained below).
+
+- `use-first` will randomly accept one of the duplicated records.
+
+<a name='fields'></a>
+
+#### `fields`
+Fields the users should fill in. 
+
+For relational records, Import support only **reference** (i.e. 1-many).
+
+```yml
+imports:
+  import-name-random: 
+    fields:
+      - name: a normal field
+      - name: a reference field
+        reference_target: the referenced table 
+        reference_field_name: the field user should fill in the CSV to idenitfy the referenced record
+        handle_duplicated_reference: throw-error or use-first
+```
+- `throw-error` will return an error. If you want to reject the whole import when there are errors, you can set `atomic: true` in the `action` setting (will be explained below).
+
+- `use-first` will randomly assign the record to one of the duplicated references.
+
+### Add the import button
+
+To add an import button to the page, you need configure it in the list view. 
+
+Example usage:
+
+Suppose now we are adding an **import** button on the `comment`'s list view.
+
+```yml
+site:
+  - type: Record
+    name: comment
+
+records:
+  comment:
+    record_type: comment
+    list:
+      actions:
+        - type: Import
+          name: import-comment # this is defined in the import section
+          label: Import
+          atomic: true 
+```
+
+- `atomic`: When `atomtic` is set to true, the whole import will be rejected if there are errors from `handle_duplicated_reference` and `handle_duplicated_identifier`. Default is false.
+
+## Exports
+
+![Skygear CMS export](/assets/cms/cms-export.png)
+
+Your users can export data from Skygear to a CSV. To do so, you need to configure the export settings and add an export button to the right page.
+
+### Configuring exports
+
+Reusing the two databases in the Configuring imports section as an example.
+
+This is what we need:
+
+1. Allow users to export all the comments
```

Comment:
Not sure if you want to mention, there may be filters in the list page. If user click export at that point, they can choose either `all records` or ``filtered records`.

Reply:
added.